### PR TITLE
fix(material-experimental/mdc-slider): correct description of slider …

### PR DESCRIPTION
**Pull request by wagnermaciel**
…harness thumb getters

### CODE REVIEW

**Review by crisbeto — APPROVED** [review]
LGTM. Maybe we should throw an error if the method is called on a non-range slider?

File: src/material-experimental/mdc-slider/testing/slider-harness.ts [hunk 1]
```diff
@@ -28,12 +28,12 @@ export class MatSliderHarness extends ComponentHarness {
       });
   }
 
-  /** Gets the start/primary thumb of the slider. */
+  /** Gets the start thumb of the slider (only applicable for range sliders). */
```
Comment (crisbeto) [inline src/material-experimental/mdc-slider/testing/slider-harness.ts:31]: I think this one applies to non-range sliders too.

**Comment by wagnermaciel** [comment]
@crisbeto I added a helpful error message and a unit test to verify that the error gets thrown correctly

**Review by crisbeto — APPROVED** [review]
LGTM

**Pull request merged**